android/KMEA/app/src/main/assets/keyboard.html

```diff
@@ -98,8 +98,8 @@
       }
 
       kbdInterface.registerStub({KN:keyboardName,KI:'Keyboard_'+internalName,KLC:langId,KL:languageName,
-        KF:kbdFile,KFont:font,KOskFont:oskFont});
-      kmw['setActiveKeyboard']('Keyboard_'+internalName,langId);
+        KF:kbdFile,KFont:font,KOskFont:oskFont,KP:package});
+      kmw['setActiveKeyboard'](package + '::Keyboard_'+internalName,langId);
```

Comment:
We need to make sure that this namespacing scheme is documented in `setActiveKeyboard` (not a part of this PR, of course).

Reply:
I've added this as a TODO item on Trello card https://trello.com/c/xO1RyzT6